packages/cli/internal/pkg/aws/cwl/stream_logs_test.go

```diff
@@ -54,7 +54,7 @@ func TestClient_StreamLogs(t *testing.T) {
 	cancel()
 	stream := client.StreamLogs(ctx, testLogGroupName)
 	event := <-stream
-	assert.Equal(t, []string{
+	assert.ElementsMatch(t, []string{
```

Comment:
The order is somewhat important. Do you mind asserting that the values at 1 and 3 are either Hello's? There is an example here: https://github.com/aws/amazon-genomics-cli/blob/main/packages/cli/internal/pkg/aws/cwl/get_logs_test.go#L92

Reply:
if order is important we should not use map, or if we use a map we should reorder it afterwards. In this case we want event that belong to the same log stream to be ordered properly. I don't thing we can guarantee concrete index of the event.

Reply:
That is what the test is doing. If you look at the errors that are associated with the test it is because the "Hola" is being shown first sometimes. The only order that matters is that "Hello" and "World" are next to each other and "Hola" is next to "Mundo". If there is an easier way to validate the ordering then please fix it.

If this test printed out "Hello", "Hola", "Mundo", "World" then it would be invalid but the change that you are making makes it valid.

Reply:
Will it be correct if it prints "World", "Hello"?

Reply:
We really should test that as well, good catch.